packages/validator/src/cli-validator/run-validator.js

```diff
@@ -109,19 +107,12 @@ async function runValidator(cliArgs, parseOptions = {}) {
       filesWithValidExtensions.push(arg);
     } else {
       unsupportedExtensionsFound = true;
-      logger.warn(
-        chalk.yellow('[Warning]') +
```

Comment:
Also removed all the "[...]" prefixes since this is now handled by the logger stuff automatically.